templates/email/bexley/waste/_email_data.html

```diff
@@ -3,7 +3,7 @@
 is_delivery_request = report.category == 'Request new container';
 is_removal_request = report.category == 'Request container removal';
 is_container_request = is_delivery_request || is_removal_request;
-is_request_enquiry = report.category == 'Request container enquiry';
+is_request_enquiry = report.category == 'Replacement bin enquiry';
```

Comment:
We'll need to rename all the existing rows in the database when this change goes through for consistency.

Reply:
I thought this would be the simpler change 😭 
But more seriously - I would need help with that, I don't think I've run live DB changes before.
Something like
```
update problem set category = 'Replacement bin enquiry' where category = 'Request container enquiry';
```
?